js/main.js

```diff
@@ -0,0 +1,21 @@
+function randomInteger(min, max) {
+  if (min >= 0 && max >= 0) {
+    return parseInt((Math.random() * (max - min) + min), 10);
+  }
+  if (max < min || max == min) {
```

Comment:
```suggestion
  if (max < min || max === min) {
```

https://up.htmlacademy.ru/javascript/22/criteries#b17

> Вместо операторов нестрогого сравнения == и !=, используются операторы строгого сравнения ===, !==.